lib/SearchForm/SearchForm.js

```diff
@@ -55,6 +57,7 @@ export function SearchForm({
                 onChangeIndex={handleChangeIndex}
                 onChange={handleChange}
                 onClear={handleClear}
+                name={fieldName}
```

Comment:
The same, please place props in correct order.

Reply:
done